pkg/certifier/attestation/attestation_vuln.go

```diff
@@ -32,7 +32,7 @@ const (
 
 // VulnerabilityStatement defines the statement header and the vulnerability predicate
 type VulnerabilityStatement struct {
-	intoto.StatementHeader
+	attestationv1.Statement
```

Comment:
in-toto-golang is being deprecated, so moving this over. Created a issue to update the other pieces. Not breaking at the moment: https://github.com/guacsec/guac/issues/2036